apache-rat-core/src/main/java/org/apache/rat/Defaults.java

```diff
@@ -76,21 +77,43 @@ public final class Defaults {
 
     private final LicenseSetFactory setFactory;
 
-
+    static {
+         URL url = Defaults.class.getResource("/org/apache/rat/default.xml");
```

Comment:
Should we extract this URI into a constant as we are using it more than once ....